docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,135 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
```

Comment:
I suggest starting with a "purpose statement" - perhaps (a very rough idea):  

This procedure describes how to display Sawtooth and network statistics with Grafana, using Docker, InfluxDB, Sawtooth configuration files, and Telegraph.